content/spin/v2/variables.md

```diff
@@ -11,60 +11,72 @@ url = "https://github.com/fermyon/developer/blob/main/content/spin/v2/variables.
 
 Spin supports dynamic application variables. Instead of being static, their values can be updated without modifying the application, creating a simpler experience for rotating secrets, updating API endpoints, and more. 
 
-These variables are defined in a Spin application manifest (in the `[variables]` section), and their values can be set or overridden at runtime by an [application variables provider](./dynamic-configuration.md#application-variables-runtime-configuration). When running Spin locally, the variables provider can be [Hashicorp Vault](./dynamic-configuration.md#vault-application-variable-provider) for secrets, [Azure Key Vault](https://azure.microsoft.com/en-us/products/key-vault), or host environment variables.
+These variables are defined in a Spin application manifest (in the `[variables]` section), and their values can be set or overridden at runtime by an [application variables provider](./dynamic-configuration.md#application-variables-runtime-configuration). When running Spin locally, the variables provider can be [Hashicorp Vault](./dynamic-configuration.md#vault-application-variable-provider) for secrets, or host environment variables.
 
 For information about configuring application variables providers, refer to the [dynamic configuration documentation](./dynamic-configuration.md#application-variables-runtime-configuration).
 
 ## Adding Variables to Your Applications
 
 Variables are added to an application under the top-level `[variables]` section of an application manifest (`spin.toml`). Each entry must either have a default value or be marked as `required = true`. “Required” entries must be [provided](./dynamic-configuration#application-variables-runtime-configuration) with a value.
 
-For example, say an application needs to access a secret. A `[variables]` section could be added to an application's manifest with one entry for a variable named `secret`. Since there is no reasonable default value for a secret, the variable is set as required with `required = true`:
+For example, say an application needs to access a bearer token for an outbound API call. A `[variables]` section could be added to an application's manifest with one entry for a variable named `token`. Since there is no reasonable default value for a secret, the variable is set as required with `required = true`. On the other hand, say the the API endpoint is known but should be configurable for A/B testing, another v`api_uri` variable could be added with a default that can be overridden. This scenario could be configured in the application manifest as follows:
```

Comment:
This feels like a lot of conditional mood ("could be added", "could be configured") - I wonder if we could make it read more positively and concretely.

```suggestion
For example, consider an application which makes an outbound API call using a bearer token. To configure this using variables, you would:

* Add a `[variables]` section to the application manifest
* Add a `token` entry for the bearer token.  Since there is no reasonable default value for this secret, set the variable as required with `required = true`.
* Add an `api_uri` variable.  The URL _is_ known, but is useful to override, for example for A/B testing. So you can give this variable a default value with `default = "http://my-api.com"`.

The resulting application manifest looks like this:
```

Maybe that is a bit _too_ definite though!

Reply:
I love it! You are a word artist.